pkg/dbconfig/meta_config.go

```diff
@@ -98,3 +107,77 @@ func (c *MetaConfig) SetInputConfig(inputConfig string) error {
 	c.InputConfig = inputConfig
 	return nil
 }
+
+func (c *MetaConfig) configureBackendStorage(spec *v1alpha1.MetaSpec, namespace string) error {
+	if etcd := spec.GetBackendStorage().GetEtcdStorage(); etcd != nil {
+		c.Backend = ptr.To("etcd_store")
+		c.StoreAddrs = etcd.GetEndpoints()
```

Comment:
Sometime etcd also need to set a password？https://github.com/bitnami/charts/blob/main/bitnami/etcd/values.yaml#L128

Reply:
It's not supported for upstream GreptimeDB. We can add it in the future.